src/EditorFeatures/VisualBasicTest/AutomaticCompletion/AutomaticLineEnderTests.vb

```diff
@@ -224,7 +226,7 @@ End Module
 <code>
 Module M
     Sub Main()
-        Dim _
+        Dim _ 
```

Comment:
why did this change?

Reply:
Merged separately: [https://github.com/dotnet/roslyn/pull/30419](https://github.com/dotnet/roslyn/pull/30419)
VSSDK update: fixing bug on their side

---
In reply to: [222523749](https://github.com/dotnet/roslyn/pull/29016#discussion_r222523749) [](ancestors = 222523749)